src/Jaeger/Sender/JaegerSender.php

```diff
@@ -52,10 +66,12 @@ class JaegerSender implements SenderInterface
      */
     public function __construct(
         AgentIf $agentClient,
+        int $maxBufferLength,
         LoggerInterface $logger = null,
         SpanToJaegerMapper $mapper = null
     ) {
         $this->agentClient = $agentClient;
+        $this->maxBufferLength = $maxBufferLength;
```

Comment:
I propose to not break backward compatibility. 
```diff
 public function __construct(
     AgentIf $agentClient,
     LoggerInterface $logger = null,
     SpanToJaegerMapper $mapper = null
 ) {
    $this->agentClient = $agentClient;
+    $this->maxBufferLength = USE_SOME_DEFAULT_CONSTANT_HERE;
     $this->logger = $logger ?? new NullLogger();
     $this->mapper = $mapper ?? new SpanToJaegerMapper();
 }
```

Then, add for this class something like:

```php
public function setMaxBufferLength($maxBufferLength)
{
    $this->maxBufferLength = $maxBufferLength;
}
```

This will allow use sender as follows:
```php
$sender = new JaegerSender($client, $this->config->getLogger());
$sender->setMaxBufferLength( $this->config->getMaxBufferLength());
```